### ISSUE TITLE
Context properties of custom Channel ignored in LoggingHandler

### ISSUE THREAD

**Issue by letmaik** [Microsoft/ApplicationInsights-Python#104]
I'm trying to set up a LoggingHandler with a few context properties but this is impossible currently:
```py
import logging
from applicationinsights import channel
from applicationinsights.logging import LoggingHandler

channel = channel.TelemetryChannel()
channel.context.properties['foo'] = 'bar'
handler = LoggingHandler(key, telemetry_channel=channel)
logging.basicConfig(handlers=[handler], format='%(message)s', level=logging.DEBUG)

logging.info("foo") # trace does not include foo property
logging.shutdown()
```

The reason this is not working is that LoggingHandler constructs a new TelemetryClient instance which then ignores the existing context from the passed channel object:
```py
class TelemetryClient(object):
    def __init__(self, instrumentation_key, telemetry_channel=None):
        ...
        self._context = channel.TelemetryContext()
        self._context.instrumentation_key = instrumentation_key
        self._channel = telemetry_channel or channel.TelemetryChannel()

    @property
    def context(self):
        return self._context
```

Shouldn't this be instead:
```py
class TelemetryClient(object):
    def __init__(self, instrumentation_key, telemetry_channel=None):
        self._channel = telemetry_channel or channel.TelemetryChannel()
        self.context.instrumentation_key = instrumentation_key

    @property
    def context(self):
        return self._channel.context
```
?

**Comment by SergeyKanzhelev** [comment]
I think a better fix will be merging contexts in [channel](https://github.com/Microsoft/ApplicationInsights-Python/blob/457c3f836a1cec010e36f89d29caae46edb3dcc7/applicationinsights/channel/TelemetryChannel.py#L83) instead of overwriting. Same as it does with [properties](https://github.com/Microsoft/ApplicationInsights-Python/blob/457c3f836a1cec010e36f89d29caae46edb3dcc7/applicationinsights/channel/TelemetryChannel.py#L104-L106).

Would you like to send a PR?

**Comment by letmaik** [comment]
I don't understand why you say overwriting, where would that happen in my code?

**Comment by SergeyKanzhelev** [comment]
not in your code. My understanding of what's going on is that `logging.info("foo")` will call channel's `write` method with the context object. So channel's context will not be used. I suggest to make property by property merge of contexts instead. So new context will have properties set by both - channel context and one passed to channel by user.

**Comment by letmaik** [comment]
Fixed in #102
